pom.xml

```diff
@@ -324,6 +325,31 @@
             </archive>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>com.diffplug.spotless</groupId>
+          <artifactId>spotless-maven-plugin</artifactId>
+          <version>${version.spotless}</version>
+          <executions>
+            <execution>
+              <id>spotless-check</id>
+              <!-- Runs in install phase to fail fast in case of formatting issues.-->
+              <phase>install</phase>
+              <goals>
+                <goal>check</goal>
+              </goals>
+            </execution>
+          </executions>
+          <configuration>
+            <encoding>UTF-8</encoding>
+            <java>
+              <toggleOffOn />
```

Comment:
What's this about? We don't have it in [sonar-iac](https://github.com/SonarSource/sonar-iac/blob/master/pom.xml#L264-L269)

Reply:
There are a couple of places with comments formatted with spaces, which I decided to keep. So I needed to turn off formatting for those parts
https://github.com/SonarSource/sonar-php/blob/1eae3e301fd0d825be7079ce8d456721fbb88279/php-checks/src/main/java/org/sonar/php/checks/HardCodedIpAddressCheck.java#L43-L62